utest/libdoc/test_libdoc.py

```diff
@@ -81,19 +81,6 @@ def test_shortdoc_with_multiline_robot_format(self):
         exp = "Writes the *message* to _the_ ``console``."
         verify_keyword_shortdoc('ROBOT', doc, exp)
 
-    def test_shortdoc_with_multiline_reST_format(self):
```

Comment:
Should be fine as long as you add docutils into utest/requirements.txt.

Reply:
@pekkaklarck should/could we add docutils to utest requirements?
Then we could run utests on KeywordDoc with reST

Reply:
We just added`utest/requirements.txt` and we certainly can add docutils there. Unfortunately it doesn't work with IronPython so it needs to be added like

```
docutils >= 0.9; platform_python_implementation != 'IronPython'
pygments; platform_python_implementation != 'IronPython'
```

and this test needs to be run only if docutils is installed. That needs to be done with something like

```python
try:
    import docutils
except ImportError:
    docutils = None

...


    if docutils:

        def test_whatever_that_needs_docutils(self):
            ...
```